src/system/utils/wordpress/rest.js

```diff
@@ -43,7 +44,7 @@ const cache = setupCache( {
  *
  * @type {AxiosInstance}
  */
-const http = axios.create( {
+export const http = axios.create( {
```

Comment:
Does this really need to be exported? That's adding it to the public api by doing that.

Reply:
Not needed anymore. Used create method of attachment from rest.js